content/realtime/channels.textile

```diff
@@ -236,6 +236,147 @@ channel.subscribe { message in
 }
 ```
 
+h4(#rewind). Subscribing to a channel and fetching history with rewind
```

Comment:
IMHO, the title feels like it's two actions like subscribe to channel and make a call to the history api. Perhaps, 'Subscribing to a channel with rewind option enabled'?